src/cephadm/cephadmlib/constants.py

```diff
@@ -12,7 +12,7 @@
 DEFAULT_GRAFANA_IMAGE = 'quay.io/ceph/grafana:9.4.12'
 DEFAULT_HAPROXY_IMAGE = 'quay.io/ceph/haproxy:2.3'
 DEFAULT_KEEPALIVED_IMAGE = 'quay.io/ceph/keepalived:2.2.4'
-DEFAULT_NVMEOF_IMAGE = 'quay.io/ceph/nvmeof:1.0.0'
+DEFAULT_NVMEOF_IMAGE = 'quay.io/ceph/nvmeof:1.2.1'
```

Comment:
This version is not yet published (pending of [this fix](https://github.com/ceph/ceph-nvmeof/pull/568) to be released).